app/src/main/java/com/globant/weatherapp/mvp/view/ActivityView.kt

```diff
@@ -0,0 +1,17 @@
+package com.globant.weatherapp.mvp.view
```

Comment:
you can put this class in a package called base inside the view package

Reply:
Done!